### ISSUE TITLE
Plugin syncs, claims files have been updated... but where are the files?

### ISSUE THREAD

**Issue by cstromblad** [renehernandez/obsidian-readwise#61]
I managed to get the plugin working a while back. But then, for some reason... no new files are added nor old files updated even though sync-notification claims files have been updated with new highlights.

The plugin is configured to sync notes into a folder I've named `/Readwise.io/`. I can add new Highlights to Readwise, manually trigger sync, get the update message... but no new files appear.

What might be wrong?

**Comment by renehernandez** [comment]
Coming back to this. Do you still have this issue? Can you provide some logs?

**Comment by cstromblad** [comment]
No, since switching to the official plugin things are working again. Thanks for reminding me to close this!
